android/app/src/main/java/labs/pooh/eaterslab/util/PlotUtils.kt

```diff
@@ -0,0 +1,162 @@
+package labs.pooh.eaterslab.util
+
+import android.content.Context
+import android.graphics.Color
+import android.view.View
+import android.widget.FrameLayout
+import io.data2viz.axis.Orient
+import io.data2viz.axis.axis
+import io.data2viz.color.Colors
+import io.data2viz.geom.Size
+import io.data2viz.scale.FirstLastRange
+import io.data2viz.scale.Scales
+import io.data2viz.scale.Tickable
+import io.data2viz.viz.*
+
+data class PlotProportion(val x: Double, val y: Double)
+
+private fun GroupNode.moveTransform(padding: Double, width: Double, index: Int, height: Double, labelSize: Double, space: Double = 0.0) {
+    transform {
+        translate(
+            x = padding + index * (width + space),
+            y = height + padding + labelSize
+        )
+    }
+}
+
+fun <T: Number> Context.horizontalBarPlot(data: List<T>, barColor: Int,
+                                  lowValueFix: Double = 1.0, proportion: PlotProportion = PlotProportion(2.0, 1.0), plotLabels: Boolean = true,
+                                  barWidth: Double = 10.0, labelColor: Int = Color.BLACK, padding: Double = 5.0, labelSize: Double = 9.0,
+                                  ticks: List<String> = listOf(), ticksIndexer: ((String) -> Int)? = null, ticksScale: Double = 4.0, ticksDistance: Double = 0.5): View {
```

Comment:
Very long lines of code, maybe change it?

Reply:
Mean spliting it in classes with default parameters or what kind of solution?